netlink-request/Cargo.toml

```diff
@@ -8,3 +8,4 @@ netlink-sys = "0.8"
 netlink-packet-core = "0.4"
 netlink-packet-generic = "0.3"
 netlink-packet-route = "0.13"
+nix = "0.25"
```

Comment:
I see that nix has many features, all of them enabled by default. Looks like we can only enable nix cargo feature called `feature` (:srug:) to save a bit of compilation time?